### PR TITLE
Library Completed

### PR DESCRIPTION
**Pull request by Aaron-RN**
Added Ability to add books to add, remove and mark books as read.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+const readBook = (e) => {
+  const { bookId } = e.target.dataset;
+  const book = myLibrary.find(book => book.id === bookId);
+  book.toggleRead();
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+  render();
+};
+/* eslint-enable no-use-before-define */
+
+const bookToHTML = (book) => `
```
Comment (bellom) [inline js/main.js]: Let separate the DOM from application logic
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        <p class="card-text">${book.desc}</p>
+        <button class="card-link btn btn-sm btn-read btn-${book.wasRead ? 'success' : 'secondary'}" data-book-id="${book.id}">
+          ${book.wasRead ? 'Mark as unread' : 'Mark as read'}
+        </button>
+        <button class="card-link btn btn-sm btn-danger btn-remove-book" data-book-id="${book.id}">Remove book</button>
+      </div>
+    </div>
+  </div>`;
+
+const errorToHTML = (error) => `
```
Comment (bellom) [inline js/main.js]: Let separate the DOM from application logic, you can create another file for DOM.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Below are part of the checklist for this project, kindly work on these few things listed below;**

**_1. The form should have `number of pages` field, user should be able to add number of pages._**
image

**_2. Kindly attend to my comments below._**


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED

Looks good Team, :+1:  everything is finely done, **please merge and move forward** 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**